modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/index/ComplexSecondaryKeyUnwrapSelfTest.java

```diff
@@ -97,7 +94,7 @@ public void testSecondaryIndexSimplePk() {
 
             executeSql("CREATE INDEX ON " + tblName + "(id, name, city)");
 
-            checkUsingIndexes(tblName, val);
+            checkUsingIndexes(tblName, val, 1, "Query with type column: " + type);
```

Comment:
Can we use the same label as for other test, like "Query shouldn't be splitted"?

Reply:
The parameter expResCnt in method should determine number items in the plan. 
I'll separate method for the two-phase plan only.